lib/sort.c

```diff
@@ -1264,6 +1263,7 @@ grn_table_sort_keys_parse_one(grn_ctx *ctx,
                                           code_end_offset);
     key->key = sliced_expr;
   }
+  key->flags = flags;
```

Comment:
Could you move this to the above line of `if (n_codes == 1 &&`?
We can reduce code range that uses `flags` by the move.

Reply:
I see.
I've fixed it in https://github.com/groonga/groonga/pull/1179/commits/7fdaf663ce31591c4ab4394905e7c03f83262a11.